src/Typewriter/Options.cs

```diff
@@ -81,5 +81,8 @@ public class Options
         [Option('t', "transform", HelpText = "Specify the URI to the XSLT that will preprocess the metadata. Overrides the" +
             "cleaning done by embeddeded typewriter.exe rules.")]
         public string Transform { get; set; }
+
+        [Option('r', "removeannotations", Default = (bool)true, HelpText = "Specify whether to keep source CSDL capability annotations. Only applicable when using generationMode Transform or TransformWithDocs.")]
```

Comment:
nit: the parameter name is "remove" but the description is "remove" and the default is true. I think this is confusion prone. Either change the parameter name, or update the description & default.